ocaml/xapi-cli-server/cli_frontend.ml

```diff
@@ -4052,7 +4052,7 @@ let rio_help printer minimal cmd =
         in
         printer (Cli_printer.PTable [recs])
     | None ->
-        D.log_backtrace () ;
+        D.log_backtrace Not_found;
```

Comment:
Why is this line here at all, as there is no exception? Leftover from the pre-find_opt days perhaps?

Reply:
You are probably right (looking through git history).
I'll remove these in a separate commit (so that the first commits largely don't change observable behaviour wrt to presence of stacktraces, i.e. if a stacktrace was logged before, it should also be logged after, but with more detail), but then the later commit will clean these up (fewer stacktraces in our logs :) )

Reply:
removed all calls to `log_backtrace` with constant arguments.